### PR TITLE
LPS-144671 Add bulk export controls to the Journal mgmt toolbar

### PR DESCRIPTION
**Pull request by adolfopa**
This PR adds bulk export support to Journal. @boton, you need to pay special attention to https://github.com/liferay-lima/liferay-portal/pull/2606/commits/f26130b36253912d35904d94534c4ccd284ae158 (the other changes are mostly backend related). In particular, I had to explicitly set the 'disabled' attribute to false, otherwise the button would be never available (that's not necessary for the other controls, no idea why).
Also, I had to use a different icon than the one in the designs, as the one used there is not available.

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:translation

**Comment by liuxingye** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -119,6 +125,17 @@ public JournalManagementToolbarDisplayContext(
 								LanguageUtil.get(httpServletRequest, "move"));
 							dropdownItem.setQuickAction(true);
 						}
+					).add(
+						dropdownItem -> {
+							dropdownItem.putData("action", "exportTranslation");
+							dropdownItem.setDisabled(false);
```
Comment (boton) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java:136]: It isn't needed on my tests

**Review by boton — COMMENTED** [review]
The frontend LGTM, only a minor comment inline.

But the backend isn't working for me at the time to import a zip.

```
2022-01-04 12:36:59.005 ERROR [http-nio-8080-exec-8][render_portlet_jsp:131] null
com.liferay.info.exception.NoSuchInfoItemException: Unable to get journal article {className=com.liferay.info.item.ClassPKInfoItemIdentifier, classPK=0}
...
```

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
With the last changes, it's working as expected 😄

**Comment by boton** [comment]
ci:forward

**Comment by boton** [comment]
@adolfopa I pushed the baseline commit

**Comment by boton** [comment]
ci:forward

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**